CMakeLists.txt

```diff
@@ -13,11 +13,18 @@ option(BUILD_APP "Build Chatterino" ON)
 option(BUILD_TESTS "Build the tests for Chatterino" OFF)
 option(USE_SYSTEM_PAJLADA_SETTINGS "Use system pajlada settings library" OFF)
 option(USE_SYSTEM_LIBCOMMUNI "Use system communi library" OFF)
-option(USE_SYSTEM_QT5KEYCHAIN "Use system Qt5Keychain library" OFF)
+option(USE_SYSTEM_QTKEYCHAIN "Use system QtKeychain library" OFF)
```

Comment:
Make sure you add a dev changelog entry for this

Reply:
Hopefully 7f8e05b is a good changelog entry for this one.